Assets/LeapMotion/Core/Scripts/VR/LeapVRTemporalWarping.cs

```diff
@@ -236,15 +261,29 @@ protected struct TransformData {
         _headTransform = transform.parent;
       }
 
+#if UNITY_EDITOR
       if (_headTransform != null && UnityEditor.PlayerSettings.virtualRealitySupported) {
         _trackingAnchor = _headTransform.parent;
       }
-    }
 #endif
 
+      deviceOffsetYAxis = _deviceOffsetYAxis;
+      deviceOffsetZAxis = _deviceOffsetZAxis;
```

Comment:
These assignments do nothing right?  Because the properties just seem to be backers for the fields themselves.

Reply:
you're right, this is vestigial now, gotta remove that.